src/models/Article.php

```diff
@@ -116,9 +123,11 @@ public function attributeLabels()
     public function ngRestAttributeTypes()
     {
         return [
+            'id' => 'text',
             'title' => 'text',
-            'teaser_text' => ['textarea', 'markdown' => true],
-            'text' => ['textarea', 'markdown' => true],
+            'alias' => 'text',
+            'teaser_text' => ['textarea', 'markdown' => false],
```

Comment:
Markdown parsing should be enabled in order to support bc.

Reply:
Unfortunately, module stops working if markdown is enabled and I've pasted long "complex" html into news body. Not sure what breaks, and it doesn't break all the time. 
What's BC? Breadcrumbs? Maybe I can revert markdown settings back to default but add module option to disable markdown?

Reply:
It will break `backward compatibility`, so all the existing news with markdown wont work anymore. so we have to keep markdown enabled.

Reply:
Oh, makes sense!

Reply:
@nadar oh, makes sense! Sorry, I did not play for many years..